app/client.js

```diff
@@ -18,22 +18,27 @@ import buildInfo from './build-info';
 import getMuiTheme from 'material-ui/styles/getMuiTheme';
 import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 
+const plugContext = (f) => () => ({
+  plugComponentContext: f,
+  plugActionContext: f,
+  plugStoreContext: f,
+});
+
 const piwik = require('./util/piwik').getTracker(config.PIWIK_ADDRESS, config.PIWIK_ID);
+
+const addPiwik = (context) => { context.piwik = piwik; }; // eslint-disable-line no-param-reassign
```

Comment:
For some reason Feedback panel does not get piwik through context:
https://github.com/HSLdevcom/digitransit-ui/blob/master/app/component/feedback/feedback-panel.cjsx#L20

It remains undefined. This is currently also broken in Master